### ISSUE TITLE
Turn off column reorder in natural_join?

### ISSUE THREAD

**Issue by Diego-Hernandez-Moodys** [WinVector/rquery#17]
Is it possible to add a flag to NOT reorder the columns in alphabetical order in the joined output?

**Comment by JohnMount** [comment]
Good point, in fact that should be the default behavior.  Probably can get to that in about a week. Thanks for the note.
